### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by magicman9318** [aclap-dev/video-downloadhelper#203] [License & Purchase requests]
### What's your issue?

I noticed purchase a wrong platform license, i want use for edge, but i using wrong browser (chrome) to purchase, can change my license to edge?? Plsssss TQ

### The 10 first digits (only 10) of your license number or email address

11dd1fc2db

### I confirm that I didn't not paste my full license number

- [x] I confirm

**Comment by jcfrog** [comment]
Hi,
Chrome and Edge premium are separate products with different prices and features.
What I can do is refund your Chrome premium so that you can purchase Edge premium if you want to.
Do you want me to do this ?
